src/Mvc/Mvc.Core/src/Infrastructure/MvcCoreMvcOptionsSetup.cs

```diff
@@ -80,6 +80,9 @@ public void Configure(MvcOptions options)
             // Set up default input formatters.
             options.InputFormatters.Add(new SystemTextJsonInputFormatter(_jsonOptions.Value, _loggerFactory.CreateLogger<SystemTextJsonInputFormatter>()));
 
+            // Media type formatter mappings for JSON
+            options.FormatterMappings.SetMediaTypeMappingForFormat("json", MediaTypeHeaderValues.ApplicationJson);
```

Comment:
Will it just no-op if this is also added by the newtonsoft json package?

Reply:
Yeah, it's just a dictionary setter: https://github.com/aspnet/AspNetCore/blob/c565386a3ed135560bc2e9017aa54a950b4e35dd/src/Mvc/Mvc.Core/src/Formatters/FormatterMappings.cs#L60